<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="horizontalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QStackedWidget" name="photoWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>429</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>10</number>
       </property>
       <widget class="QWidget" name="starta">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/MenuScreen/images/489ceccb15bbd05ea614de990118cde0.gif);</string>
        </property>
       </widget>
       <widget class="QWidget" name="wordleRoom">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Pictures/Wordle-235-February-9-2022-Answer-2.webp);</string>
        </property>
       </widget>
       <widget class="QWidget" name="b">
        <property name="styleSheet">
         <string notr="true">
border-image: url(:/images/other bar.gif);</string>
        </property>
       </widget>
       <widget class="QWidget" name="c">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/bar.gif);</string>
        </property>
       </widget>
       <widget class="QWidget" name="d">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/alleyway 2.jpg);</string>
        </property>
       </widget>
       <widget class="QWidget" name="f">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/danger alley.jpg);</string>
        </property>
       </widget>
       <widget class="QWidget" name="g">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/end screen.gif.crdownload);</string>
        </property>
       </widget>
       <widget class="QWidget" name="e">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/store.jpg);</string>
        </property>
       </widget>
       <widget class="QWidget" name="h">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/store.gif);</string>
        </property>
       </widget>
       <widget class="QWidget" name="gameOver">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/game over.jpg);</string>
        </property>
       </widget>
       <widget class="QWidget" name="Map">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/map.png);</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="westButton">
       <property name="styleSheet">
        <string notr="true">font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>West</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="eastButton">
       <property name="styleSheet">
        <string notr="true">font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>East</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="northButton">
       <property name="styleSheet">
        <string notr="true">
font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>North</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="southButton">
       <property name="styleSheet">
        <string notr="true">font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>South</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>789</x>
      <y>-1</y>
      <width>271</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="textWidget">
       <widget class="QWidget" name="a">
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;You wake up and as you take in your surroundings you realise your in Zork city, where crime is rampant and everything is opened with keys and oddly placed Wordle puzzles. You set off, finding a map on the ground next to you.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="wordlePage">
        <widget class="QTextEdit" name="textEdit_11">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;You stumble upon a large machine with an intricate lock. After inspection you realise that a wordle is needed to open it. Will you try solve the wordle or continue ahead.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="b_2">
        <widget class="QTextEdit" name="textEdit_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;As you walk out into the open street, you see a bar in front of you. To the right a purple light emits from a 24/7 store and to the left a dark alley calls to you.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="c_2">
        <widget class="QTextEdit" name="textEdit_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;A heavy cloud of smoke fills the bar and a strong scent of whiskey fills your nostrils. Looking across the room you see a large door with an intricate lock on it. According to a sign next to it, this is the entrance to the rest of the city and one must find a key in order to gain access. Maybe the alleyway or store might have the key.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="d_2">
        <widget class="QTextEdit" name="textEdit_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;This alleyway has a strong stench to it, making you wretch. There's almost nothing of note to it, apart from a few boxes and a large dumpster. Maybe there might be something of value.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="f_2">
        <widget class="QTextEdit" name="textEdit_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;As you turn the corner you feel an instant sense of danger. Two men stand in front of you with murderous intent with their hoods up. It's best to turn around right now, unless you're feeling lucky&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="g_2">
        <widget class="QTextEdit" name="textEdit_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;A wave of relief floods over you as you breath a breath of fresh air. You've unlocked the rest of the city, what you do now is up to you. To finish your journey, press the Quit button.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="e_2">
        <widget class="QTextEdit" name="textEdit_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;A purple hue emits from the store, looking around this seems to be the end of this part of the city. Should you turn around or head inside?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="h_2">
        <widget class="QTextEdit" name="textEdit_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;The store clerk looks up at you as you enter. An assortment of high tech items are displayed throughout the room with prices way above your salary. You wonder why you are wasting your time here.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="gameOver1">
        <widget class="QTextEdit" name="textEdit_9">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;You stand forward ready to have a honourable fight, but as soon as you do you slip on a banana peel and fall face first into some trash bags. This is when the two men descend on you, robbing you and putting a knife into your back for good measure. Your world fades to black and your dreams of leaving this alley are long gone...&lt;/span&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; text-decoration: underline;&quot;&gt;press Quit to end the game.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="keyTaken">
        <widget class="QTextEdit" name="textEdit_10">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>451</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;After searching through piles of rubbish, you feel the metal handle of a key! Putting it in your pocket, you get ready to head back to the bar.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>600</y>
      <width>271</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="Map_2">
       <property name="styleSheet">
        <string notr="true">font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Map</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="itemWidget">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>459</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>-1</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/key.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="interactWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>459</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="fpage">
     <widget class="QPushButton" name="fight">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>269</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Fight</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="startpage"/>
    <widget class="QWidget" name="keypage">
     <widget class="QPushButton" name="Search">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>269</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Opendoor">
     <widget class="QPushButton" name="Unlock">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>269</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Unlock</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Solve">
     <widget class="QPushButton" name="SolveWordle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>269</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Solve Wordle</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>650</y>
      <width>269</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 16pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="itemWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>460</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true">image: url(:/images/knife.jpg);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
